plugins/modules/keycloak_user_federation.py

```diff
@@ -85,6 +85,12 @@
             - parentId
         type: str
 
+    removeUnspecifiedMappers:
```

Comment:
Please avoid camel case for module options (that don't correspond to Keycloak properties that are camel case):
```suggestion
    remove_unspecified_mappers:
```